bokehjs/src/coffee/common/selector.coffee

```diff
@@ -10,8 +10,10 @@ class Selector extends HasProps
     @set('timestamp', new Date(), {silent: silent})
     @set('final', final, {silent: silent})
     if append
+      if @get('indices').get_model()
+        indices.get_model = @get('indices').get_model
+        indices.get_view = @get('indices').get_view
       indices['0d'].indices =  _.union(@get('indices')['0d'].indices, indices['0d'].indices)
-      indices['0d'].glyph =  @get('indices')['0d'].glyph or indices['0d'].glyph
       indices['1d'].indices =  _.union(@get('indices')['1d'].indices, indices['1d'].indices)
       indices['2d'].indices =  _.union(@get('indices')['2d'].indices, indices['2d'].indices)
```

Comment:
@bryevdv what actually happens here? Does this combine the indices from multiple "selections" on the same glyph, or of multiple glyphs? If the former, my earlier comment about selecting multiple lines can be ignored.